src/sentry/services/hybrid_cloud/organizationmember_mapping/model.py

```diff
@@ -25,14 +25,6 @@ class RpcOrganizationMemberMapping(RpcModel):
 
 
 class RpcOrganizationMemberMappingUpdate(RpcModel):
-    """
```

Comment:
This comment is longer true -- it's a true replacement call.

Reply:
Looks like it hasn't been true for some time. An "omitted key" is possible with a `TypedDict` but not a Pydantic model. Did this get changed from a `TypedDict` at some point?